ipaserver/install/dsinstance.py

```diff
@@ -199,6 +199,7 @@ def get_all_external_schema_files(root):
 RootDNPwd= $PASSWORD
 InstallLdifFile= /var/lib/dirsrv/boot.ldif
 inst_dir=   /var/lib/dirsrv/scripts-$SERVERID
+ldapifilepath=/var/run/slapd-$SERVERID.socket
```

Comment:
This should use paths.SLAPD_INSTANCE_SOCKET_TEMPLATE.

Reply:
ACK